hashtable/chaining/hashmap_test.go

```diff
@@ -0,0 +1,69 @@
+package chaining
+
+import (
+	"testing"
+)
+
+func TestHashMap(t *testing.T) {
```

Comment:
Not sure about how test cases are written in Go but in other languages, each method will test only 1 test case.

Reply:
I'm not sure neither. I'm assuming that the `TestHashMap` is like a `describe` function, and `t.Run` is like small test cases. How about this @thanks173 ?